src/core/MRHyperlink.js

```diff
@@ -0,0 +1,35 @@
+import { MRTextEntity } from './MRTextEntity';
+
+/**
+ * @class A
+ * @classdesc 3D representation of a hyperlink. `mr-a`
+ * @augments MRDivEntity
+ */
```

Comment:
you have to include { MRDivEntity } as a normal include at the top of the file for the documentation to be happy

Reply:
(the warning shows up in the 'view files' version of the pr for refernce)

Reply:
ah meant to be MRTextEntity but forgot to change it.